.gitignore

```diff
@@ -3,20 +3,14 @@ node_modules
 tmp
 
 # ignore js, map, and css in element dirs
+elements/*/_temp
 elements/*/*.js
 elements/*/*.map
 elements/*/*.css
 # but don't ignore tooling files
 !elements/*/gulpfile.js
 !elements/*/rollup.config.js
 
```

Comment:
Yay, thanks for cleaning up the crumbs left behind by `themes`.

Reply:
No problemo!